Telerik.Examples.RazorPages/Telerik.Examples.RazorPages/Pages/Captcha/CaptchaTagHelper.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@page
+@model Telerik.Examples.RazorPages.Pages.Captcha.CaptchaTagHelperModel
+
+@using Telerik.Examples.RazorPages.Models
+@using Kendo.Mvc.UI
+
+@inject Microsoft.AspNetCore.Antiforgery.IAntiforgery Xsrf
+
+@{
+    var token = Xsrf.GetAndStoreTokens(HttpContext).RequestToken;
+}
+
+<form>
```

Comment:
Use the [Validator ](https://demos.telerik.com/aspnet-core/validator) to display the error message when the Captcha is invalid/invalid:
```
<form id="exampleForm"  kendo-validator="true">
  ...
</form>
```